src/main/java/intellistart/interviewplanning/controllers/dto/BookingDto.kt

```diff
@@ -0,0 +1,24 @@
+package intellistart.interviewplanning.controllers.dto
+
+import intellistart.interviewplanning.model.booking.Booking
+
+/**
+ * Booking data transfer object from RestController.
+ */
+data class BookingDto(
+    var interviewerSlotId: Long = 0,
+    var candidateSlotId: Long = 0,
+    var from: String = "",
+    var to: String = "",
+    var subject: String = "",
+    var description: String = ""
+)
+
+fun Booking.toDTO():BookingDto = BookingDto(
```

Comment:
```suggestion
fun Booking.toDto(): BookingDto = BookingDto(
```
fix formatting here and in similar places